.github/workflows/lte-integ-test-bazel-magma-deb.yml

```diff
@@ -51,11 +52,15 @@ jobs:
 
       - uses: actions/setup-python@13ae5bb136fac2878aff31522b9efb785519f984 # pin@v4.3.0
         with:
-          python-version: '3.8.10'
+          python-version: '3.10.12'
       - name: Install pre requisites
         run: |
           pip3 install --upgrade pip
           pip3 install ansible fabric jsonpickle requests PyYAML firebase_admin
+          sudo apt update && sudo apt install virtualbox
```

Comment:
Wait nested virtualization is enabled on Ubuntu nodes?

Reply:
Yes, @tmdzk , nested virtualization is indeed enabled on the Ubuntu nodes. This enables us to install VirtualBox within our workflow, allowing us to utilize virtualization capabilities within our virtual environments.

Reply:
That's a great idea, I'm all in!